gui/src/pages/NewProject.svelte

```diff
@@ -45,7 +48,11 @@
                 .slice(0, search.length) === search.toLocaleLowerCase()) as template}
           <button
             on:click={() => {
-              router.push(`#/new-project/${template.name}`);
+              router.push(
+                `#/new-project/${template.name}${
```

Comment:
Even though we control the template name, we should code defensively here and use an encodeURIComponent on it.